packages/url-state-provider/src/index.ts

```diff
@@ -255,7 +267,7 @@ function informListener(stateID, newState, oldState) {
   listener(newState)
 }
 
-function onGlobalChange(callback) {
+function onGlobalChange(callback: any): any {
```

Comment:
Same here, may be there is a way to avoid `any`?

Reply:
This parts are more your code so i was a bit less specific, because I understand less. So I guess you are right - in the end i can take the same type like in superstate.